fava/api/__init__.py

```diff
@@ -626,3 +626,10 @@ def account_open_metadata(self, account_name):
             if isinstance(posting, Open):
                 return posting.meta
         return {}
+
+    def sidebar_links(self):
+        # 2016-04-01 custom "fava-sidebar-link" "Income 2014" "/income_statement?time=2014"  # noqa
+        return [(entry.values[0].value, entry.values[1].value)
```

Comment:
This runs through all entries once on each page load which is horribly inefficient for such a small feature.